skeleton/local-deploy/deployment/README.md

```diff
@@ -0,0 +1,78 @@
+# Deploying Locally
```

Comment:
Having a readme in the root of the directory to point to this would be good (or having this be the readme)

Reply:
I considered moving it to the root of the repo but thought that might interfere if the application ever were to come with a readme itself. Right now the samples don't have one but I would think some would?

Reply:
Yeah, that's fair. I wonder if there's something we can inject into one of the steps for the template to tell the user to go to `deployment`?

Reply:
As part of the output steps of the template it generates a series of links (for local just the one to the repo), I'm sure we could easily add one there for the deployment dir

Reply:
I believe the link I added at the end is going to show up if a user doesn't choose local deployment so i am looking into how to resolve that, after that's done this should be good for a re-review